patches/api/0327-Expose-tracked-entities.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SuperKooks <superkooks@superkooks.com>
+Date: Wed, 4 Aug 2021 18:18:52 +1000
+Subject: [PATCH] Expose tracked entities
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 5e8814cc317a705eaf8bdd9f3876a5366c0a0226..1a63144cd001a6b9d1ddf37ba795d7fb2e94c9ee 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -2,6 +2,7 @@ package org.bukkit.entity;
+ 
+ import java.net.InetSocketAddress;
+ import java.util.Set; // Paper
++import java.util.Collection; // Paper
+ import java.util.UUID;
+ import com.destroystokyo.paper.ClientOption; // Paper
+ import com.destroystokyo.paper.Title; // Paper
+@@ -2050,6 +2051,16 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+     Set<Player> getTrackedPlayers();
+     // Paper end
+ 
++    // Paper start
++    /**
++     * Gets entities within this player's tracking range (that the player's client can "see")
++     * @param entityClass The class of entities to filter for
++     * @return The filtered set of entities
++     */
++    @NotNull
++    <T extends org.bukkit.entity.Entity> Collection<? extends T> getTrackedEntities(@NotNull Class<? extends T> entityClass);
```

Comment:
Is this not still wrong? The parameter should just be `Class<T>` instead unless I'm missing something.

Reply:
The `<? extends T>` was part of @Proximyst 's requested changes.

Reply:
For the return type yes, not for the parameter.

Reply:
Right. @Proximyst's change included the parameter. I'll change it back.